README.md

```diff
@@ -88,11 +90,11 @@ The above exception handler automatically catches and sends all uncaught excepti
 
 For an actual usage example, check out the sample application in the **app** module of this project
 
-## Raygun and ProGuard
+## Raygun and ProGuard/R8
 
 ### General
 
-ProGuard is a free Java tool for obfuscation, class file shrinking, optimizing and pre-verifying. When enabling ProGuard in a native Android application that also uses Raygun, the obfuscation feature requires a bit of attention. By default, your obfuscated class and method names will show up in the stacktraces of exception/error reports submitted to Raygun. This makes the stacktraces difficult to read when looking into the cause of the issues.
+ProGuard and R8 are tools for obfuscation, class file shrinking, optimizing and pre-verifying. When enabling ProGuard in a native Android application that also uses Raygun, the obfuscation feature requires a bit of attention. By default, your obfuscated class and method names will show up in the stacktraces of exception/error reports submitted to Raygun. This makes the stacktraces difficult to read when looking into the cause of the issues.
```

Comment:
Is it no longer free or is R8 not free bu ProGuard is?

Reply:
The `free` comment doesn't really make sense in this context and is probably still in the doc from before I even took it over :). It comes with Android Studio as-is anyway and is now - depending on what AS folks use - either ProGuard or R8.